snc.sh

```diff
@@ -227,31 +227,57 @@ function create_pvs() {
 # This follows https://blog.openshift.com/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms/
 # in order to trigger regeneration of the initial 24h certs the installer created on the cluster
 function renew_certificates() {
-    # Get the cli image from release payload and update it to bootstrap-cred-manager resource
-    cli_image=$(${OC} adm release -a ${OPENSHIFT_PULL_SECRET_PATH} info ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE} --image-for=cli)
-    ${YQ} write kubelet-bootstrap-cred-manager-ds.yaml.in spec.template.spec.containers[0].image ${cli_image} >kubelet-bootstrap-cred-manager-ds.yaml
+    # Create the unsupported cert rotation config for a day (validity will increase by 30 times so it will become 30days)
+    ${OC} create -n openshift-config configmap unsupported-cert-rotation-config --from-literal='base=86400s'
 
-    ${OC} apply -f kubelet-bootstrap-cred-manager-ds.yaml
-    rm kubelet-bootstrap-cred-manager-ds.yaml
+    # Delete the pods to have cert rotation
+    ${OC} delete pods --all -A --force --grace-period=0
```

Comment:
Here I guess you want to delete pods that load the configmap, not all of them. That's fine but maybe it can save some time.

Consumers:
cluster-kube-apiserver-operator
cluster-kube-controller-manager-operator 

Am I wrong? (Don't bother changing this, I am just trying to understand)

Reply:
@guillaumerose I am not sure if only those are the consumers but this is something I got from the openshift QE folks so used this as it is but we can try that as separate experiment.